src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -144,6 +144,8 @@ static void uct_ib_mlx5_devx_klm_entry_set(void **klm_p, size_t klm_idx,
               mr->addr, mr->length, mr->lkey);
     UCT_IB_MLX5DV_SET64(klm, klm, address, (uintptr_t)address);
     UCT_IB_MLX5DV_SET(klm, klm, mkey, mr->lkey);
+    UCT_IB_MLX5DV_SET(klm, klm, byte_count, mr->length);
```

Comment:
After discussion with @Artemy-Mellanox it was decided that we need to set `UCT_IB_MD_MAX_MR_SIZE` as `byte_count` in single-thread case and `chunk_size` in multi-threaded since it should be the same with `entity_size` ksm argument.